### DISCUSSION TITLE
Web interface access with 'hostname.local' address instead of IP

### DISCUSSION THREAD

**Discussion by Quint097** [outdoorbits/little-backup-box#299] [Ideas]
I would like to be able to access the web interface using a address like 'hostname.local' instead of the IP, like as with the Octoprint web interface. this would simplify the setup and remove the need to search for the IP. Is it possible to do this?

**Comment by outdoorbits** [comment]
That's possible of course, but it's not an issue about lbb. If you want to do so, you have to configure this in your router. In your local network, usually your router will act as your name server (DNS). Maybe your router allows to configure local names.

**Comment by Quint097** [comment]
Okay, I'll look into that. 

Edited: Seems that it already works with on additional changes. you can just go to 'http://hostname/', where the hostname is what you've set when installing OS (default: raspberrypi).
